app/src/main/java/org/oxycblt/auxio/music/Music.kt

```diff
@@ -349,13 +349,15 @@ interface Playlist : MusicParent {
  * @author Alexander Capehart (OxygenCobalt)
  */
 class SortName(name: String, musicSettings: MusicSettings) : Comparable<SortName> {
-    private val number: Int?
     private val collationKey: CollationKey
     val thumbString: String?
 
     init {
         var sortName = name
         if (musicSettings.intelligentSorting) {
+            // Strip any quotes, dots, and open parentheses from the beginning
+            sortName = sortName.replace(Regex("""^['".(]+"""), "")
```

Comment:
I think there is a built-in escape sequence for any kind of punctuation, which should make this faster and more localizable.